### DISCUSSION TITLE
UTM for Home Hacking Lab

### DISCUSSION THREAD

**Discussion by Sn1ckleFr1tz** [utmapp/UTM#4598] [UTM Help (macOS)]
Is there any way to create a secure home Hacking lab on Apple Silicon (M1) using UTM? I need it for my Pentesting Course.How can i isolate my environment from outside using UTM (what should be the network settings for each machine) ?  

(ı need 2 machines 1 main (kali) 1 vulnerable.

- I installed kali linux via UTM using visualisation.
- I installed a vulnerable machine (in this case I must use Metasploitable2 for the course requirements, which installed using qemu to convert virtual disk images to .qcow2).

The problem is network settings. (In virtualbox you can select Internal Network but virtualbox for apple silicone is on beta and doesn't useful. 

(Other tried vmware and parallels i can run kali but i am not sure how to create a lab and isolate neither network settings, also metasploitable 2 is not m1 friendly,I needed to use qemu emulation UTM, but I am not sure about will it work on wmvare fusion tech display and paralels.

**Comment by ManuW** [comment]
I'd like to start reading my Hacking book this winter (again, second try or so...) and need the same setup. As I started to read the book, I used VirtualBox and the setup was easy.

Now, I have a M1 system and it is a little bit complicated or I'm too stupid for it... 

I have a virtual machine with Kali (ARM, Apple virtualization) running and second one, Metasploitable 2, running using QEMU. The QEMU machine has its own network and is isolated from the host system.

I'm able to connect the Kali vm to the other network. I simple move the interface from the kali-bridge to the other bridge. 

Commandos I used on macOS:

```sh
sudo ifconfig bridge101 deletem vmenet1
sudo ifconfig bridge100 addm vmenet1
```

Then I have access to the metasploit machine from the Kali machine. (Maybe I run `nmcli device up eth0` on Kali.)

If I add an additional network interface to the Kali vm, then I can use only one interface at the same time! I do not understand this?¿? I'd like to have one interface be connected to the msp. network and the other my host system. Has somebody an idea?
